### PR TITLE
Fix wstrust endpoint selection

### PR DESCRIPTION
**Pull request by omercs**
Issue #41
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/AzureAD/azure-activedirectory-library-for-java/pull/43%23discussion_r35456959%22%2C%20%22https%3A//github.com/AzureAD/azure-activedirectory-library-for-java/pull/43%23issuecomment-125768457%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/AzureAD/azure-activedirectory-library-for-java/pull/43%23issuecomment-125768457%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Ping...%22%2C%20%22created_at%22%3A%20%222015-07-28T22%3A14%3A14Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/466805%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/omercs%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%200eef3d4e805abdfdc257d474400421e7ef96f665%20src/main/java/com/microsoft/aad/adal4j/AuthenticationContext.java%2032%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/AzureAD/azure-activedirectory-library-for-java/pull/43%23discussion_r35456959%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Fixed%20tab%20format%20for%20the%20file%22%2C%20%22created_at%22%3A%20%222015-07-24T19%3A19%3A27Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/466805%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/omercs%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20src/main/java/com/microsoft/aad/adal4j/AuthenticationContext.java%3AL62-913%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull 0eef3d4e805abdfdc257d474400421e7ef96f665 src/main/java/com/microsoft/aad/adal4j/AuthenticationContext.java 32'></a> &nbsp;<b><a href='https://github.com/AzureAD/azure-activedirectory-library-for-java/pull/43#discussion_r35456959'>File: src/main/java/com/microsoft/aad/adal4j/AuthenticationContext.java:L62-913</a></b>
- <a href='https://github.com/omercs'></a> Fixed tab format for the file
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/AzureAD/azure-activedirectory-library-for-java/pull/43#issuecomment-125768457'>General Comment</a></b>
- <a href='https://github.com/omercs'></a> Ping...

<a href='https://www.codereviewhub.com/AzureAD/azure-activedirectory-library-for-java/pull/43?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/AzureAD/azure-activedirectory-library-for-java/pull/43?approve=1'></a>&nbsp;<a href='https://github.com/AzureAD/azure-activedirectory-library-for-java/pull/43'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by brandwe** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/AzureAD/azure-activedirectory-library-for-java/pull/43'>Refresh</a>.<a href='#crh-update-none'></a>

File: src/main/java/com/microsoft/aad/adal4j/AuthenticationContext.java [hunk 1]
```diff
@@ -62,852 +62,851 @@
  */
 public class AuthenticationContext {
 
-	private final Logger log = LoggerFactory
-			.getLogger(AuthenticationContext.class);
-
-	private final AuthenticationAuthority authenticationAuthority;
```
Comment (omercs) [inline src/main/java/com/microsoft/aad/adal4j/AuthenticationContext.java]: Fixed tab format for the file

**Comment by omercs** [comment]
Ping...

File: src/main/java/com/microsoft/aad/adal4j/WSTrustRequest.java [hunk 2]
```diff
@@ -58,37 +59,118 @@
...
+        //  <s:Header>
+        //      <wsa:Action s:mustUnderstand='1'>http://schemas.xmlsoap.org/ws/2005/02/trust/RST/Issue</wsa:Action>
+        //      <wsa:To s:mustUnderstand='1'>https://corp.sts.microsoft.com:443/adfs/services/trust/2005/windowstransport</wsa:To>
+        //      <wsa:MessageID>1303795308</wsa:MessageID>-<wsse:Security>-<wsu:Timestamp Id="Timestamp"><wsu:Created>2011-04-26T05:21:50Z</wsu:Created><wsu:Expires>2011-04-26T05:26:50Z</wsu:Expires></wsu:Timestamp></wsse:Security></s:Header>-<s:Body>-<wst:RequestSecurityToken Id="RST0"><wst:RequestType>http://schemas.xmlsoap.org/ws/2005/02/trust/Issue</wst:RequestType>-<wsp:AppliesTo>-<wsa:EndpointReference><wsa:Address>urn:federation:MicrosoftOnline</wsa:Address></wsa:EndpointReference></wsp:AppliesTo><wst:KeyType>http://schemas.xmlsoap.org/ws/2005/05/identity/NoProofKey</wst:KeyType></wst:RequestSecurityToken></s:Body></s:Envelope>
+        messageBuilder
+                .append(String
+                        .format("<s:Envelope xmlns:s='http://www.w3.org/2003/05/soap-envelope' xmlns:a='http://www.w3.org/2005/08/addressing' xmlns:u='%s'>"
```
Comment (aj-michael) [inline src/main/java/com/microsoft/aad/adal4j/WSTrustRequest.java]: From http://specs.xmlsoap.org/ws/2005/02/trust/WS-Trust.pdf,
xmlns:a should be http://schemas.xmlsoap.org/ws/2004/08/addressing

**Pull request merged**